README.md

```diff
@@ -40,28 +40,29 @@
 - JSX
 - CSS
 - React JS
+- Redux
 
 <!-- Features -->
 
 ### Key Features <a name="key-features"></a>
 
--[x] **Rockets**
--[x] **Dragons**
--[x] **Missions**
--[x] **Profile**
+-[x] **Serach coin**
+-[x] **Crypto coins list**
+-[x] **Crypto coins details**
+
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LIVE DEMO -->
 
-## 🚀 Live Demo <a name="live-demo"></a>(https://spacex-3bz6.onrender.com/)
+## 🚀 Live Demo <a name="live-demo"></a>(Coming soon)
 
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- Loom Walkthrough -->
 ## LOOM PROJECT PRESENTATION <a name="loom-demo"></a>
 
-- [Loom Walkthrough Link](coming soon)
+- [Loom Walkthrough Link](https://www.loom.com/share/55e4214003fe47449424e4eb5f6b408b)
```

Comment:
- [ ] Great touch on your video description. However, there should be a live demo of your app. Please deploy your app using one of these (Render, Netlify, or GitHub Pages) and add a live demo link in readme.

Reply:
- [ ] One more, I think you want to use another title here right? Link to highlight: [link](https://github.com/Bigizi/Metrics-react-app/tree/dev#:~:text=Book%2DStore%2DReact%20Project)